index.html

```diff
@@ -161,7 +161,7 @@ <h2>Uber Navigation</h2>
                         <li>css</li>
                         <li>javascript</li>
                     </ul>
-                    <button class="project-btn" type="button">See Project</button>
+                    <button class="project-btn btn" type="button" data-btn-id="4">See Project</button>
```

Comment:
Kindly note that the popup follows the design for both desktop and mobile versions from the chosen template in Figma (e.g., font, colours, images, tests, margins). 👌✔
- Figma desktop design
![screenshot (1)](https://user-images.githubusercontent.com/44624138/141010577-4aa25719-e8af-4f4a-8b98-3155aec18460.png)

Reply:
Please, can you explain more what do you mean?